cmd/config/olm/clusterextension.yaml

```diff
@@ -0,0 +1,12 @@
+apiVersion: olm.operatorframework.io/v1
+kind: ClusterExtension
+metadata:
+  name: {{ .NAME_PREFIX }}-{{ .Iteration }}
+spec:
+  description: "Benchmark extension for testing"
+  displayName: "BenchmarkExtension{{ .Iteration }}"
+  installMode: AllNamespaces
+  priority: -100
+  catalog: {{ .CATALOG_NAME_PREFIX }}{{ .Iteration }}
+  package: {{ .PACKAGE_NAME }}
```

Comment:
The issue here is that the `ClusterExtension` is cluster scope, not namespace. So, we have to install different packages, not the same one. You can refer to https://github.com/openshift/openshift-tests-private/tree/master/test/extended/operators/stress/manifests/config/pkg-ins-v1